src/Symfony/Component/HttpKernel/Debug/FileLinkFormatter.php

```diff
@@ -23,6 +23,8 @@
  */
 class FileLinkFormatter implements \Serializable
 {
+    private static $routerCalled = false;
```

Comment:
this relies on global state to workaround the issue.
better find another way...

Reply:
Thanks for the feedback, I close this PR and and perhaps I'll open an other PR with a better solution :+1: